Styles/root.css

```diff
@@ -0,0 +1,23 @@
+:root {
+
+    --primary-font: 'Montserrat', sans-serif;
+    --secondary-font: 'Russo One', sans-serif;
+    --tertiary-font: 'Satisfy', cursive;
+
+    --ecom-primary: #C7010E;
+    --ecom-secondary: #DA291C;
+
+
+    --white: #FFFFFF;
+    --black: #000000;
+    --bright-red: #FF0000;
+
+
+
+    --homecard-shadow: rgba(14, 30, 37, 0.12) 0px 2px 4px 0px, rgba(14, 30, 37, 0.32) 0px 2px 16px 0px;
+    --imageoverlay-gradient: rgba(0, 0, 0, 0) linear-gradient(to bottom, rgba(0, 0, 0, 0) 0px, rgba(0, 0, 0, 0.6) 100%) repeat 0 0;
+}
```

Comment:
please use code formatter like prettier to format your code for better readability.
also it will remove this unwanted spaces